docs/source/usage/examples.rst

```diff
@@ -3,7 +3,9 @@
 All Examples
 ============
 
-The full list of examples is contained in our ``examples/`` folder:
+The full list of example scripts shown below is also contained in our ``examples/`` folder.
+
+Example data sets can be downloaded from: `github.com/openPMD/openPMD-example-datasets <https://github.com/openPMD/openPMD-example-datasets>`_. The following command will automatically install those into `samples/` on Linux and OSX: `curl -sSL https://git.io/JewVw | bash`
```

Comment:
I forgot to tell you that verbatim quotes in `.rst`, contrary to `.md` files, need double quotes:
```suggestion
Example data sets can be downloaded from: `github.com/openPMD/openPMD-example-datasets <https://github.com/openPMD/openPMD-example-datasets>`_.
The following command will automatically install those into ``samples/`` on Linux and OSX: ``curl -sSL https://git.io/JewVw | bash``
```

Reply:
Thank you Dr Hübl for the help with the configuration and the correction of the quotes!!